### PR TITLE
A4A > Agency Tier: Fix the celebration modal overflow logic on mobile screens.

### PR DESCRIPTION
**Pull request by yashwin**
Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/1367

## Proposed Changes

This PR fixes the celebration modal UI on mobile devices.

## Why are these changes being made?

* To fix the celebration modal UI on mobile devices.

## Testing Instructions

1) Update your agency tier to Emerging Partner using the MC tool > Open the A4A live link > Go to the Overview page.
2) Switch to mobile view using the dev tool > Use different mobile views and verify that all devices look good. Also, verify that the tablet view and desktop view look good. You should see a faded horizontal div when the content is overflowing, which disappears when scrolled to the bottom.

Screenshot 2024-10-28 at 8 49 25 AM

Screenshot 2024-10-28 at 8 49 08 AM

3) Update your agency tier to Agency Partner using the MC tool > Refresh the overview page > Repeat step 2 and verify everything works as expected.

4) Update your agency tier to Pro Agency Partner using the MC tool > Refresh the overview page > Repeat step 2 and verify everything works as expected.

Screenshot 2024-10-28 at 9 15 52 AM

Screenshot 2024-10-28 at 9 16 52 AM

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/a8c-for-agencies/sections/agency-tier/lib/get-agency-tier-info.tsx [hunk 1]
```diff
@@ -144,7 +144,6 @@ const getAgencyTierInfo = (
 							"Advanced sales training sessions at request to sharpen your team's expertise."
 						),
 						translate( "Access to pre-qualified leads provided by Automattic's sales teams." ),
-						translate( "Access to pre-qualified leads provided by Automattic's sales teams." ),
```
Comment (yashwin) [inline client/a8c-for-agencies/sections/agency-tier/lib/get-agency-tier-info.tsx:147]: Remove the duplicate line item

**Comment by jkguidaven** [comment]
In the mobile view, I believe it’s better to have the CTA button span the entire width for easier reach.

Screenshot 2024-10-28 at 12 41 51 PM

**Comment by jkguidaven** [comment]
This isn't related to the mobile view, but it's an area for improvement that we can address in a separate PR.

Could we align the scrollbar to the right so it doesn't look weird? We can eliminate the gap in the main container and simply add padding to the text container.

Screenshot 2024-10-28 at 12 44 19 PM

**Review by jkguidaven — APPROVED** [review]
It works as expected and has improved the mobile viewing experience. I have a few comments, but they are non-blockers.

**Comment by yashwin** [comment]
Thanks for the review, @jkguidaven!

I have created an issue to address this: https://github.com/Automattic/automattic-for-agencies-dev/issues/1370

**Pull request merged**